react/src/components/molecules/AccordionItem/style.scss

```diff
@@ -0,0 +1,181 @@
+@import "global";
+@import 'ma-typography';
+@import 'ma-container';
+
+.ma__accordion {
+
+  &-item {
+    border: 2px solid $c-gray-dcdcdc;
+    &--borderless{
+      border: none;
+    }
+    &--secondary{
+     .ma__accordion-content__body {
+          padding-left: 35px;
+      }
+    }
+  }
+
+  &-header {
+
+    &__icon--secondary {
+      font-size: 2.5rem;
+      line-height: .65em;
+      position: absolute;
+        left: 5px;
+        top: 5px;
+      text-align: center;
+      overflow: hidden;
+      transform-origin: center center;
+      transition: transform .2s ease;
+      fill: $c-theme-secondary;
+      transform: rotate(-90deg);
+    }
+
+    &__icon--secondary + &__title--secondary {
+      padding-left: 0px;
+    }
+
+    &__button {
+      border: none;
+      display: block;
+      padding: 20px 40px;
+      position: relative;
+      text-align: left;
+      width: 100%;
+      background-color: $c-bg-section;
+
+      &--secondary {
+        display: block;
+        position: relative;
+        width: 100%;
+        text-align: left;
+        border: none;
+        padding-left: 35px;
+        &.is-open {
+          .ma__accordion-header__icon--secondary {
+            transform: rotate(0deg);
+          }
+        }
+      }
+
+      /* focus specific transitions */
+      -webkit-transition: border-color .15s ease-in-out, -webkit-box-shadow .15s ease-in-out;
+      transition: border-color .15s ease-in-out, -webkit-box-shadow .15s ease-in-out;
+      -o-transition: border-color .15s ease-in-out, box-shadow .15s ease-in-out;
+
+
+      &:focus{
+        -webkit-box-shadow: 0 0 0 2px #9bdaf1;
+        box-shadow: 0 0 0 2px #9bdaf1;
+      }
+      &--trans {
+        &.is-open {
+          background-color: transparent;
+        }
+      }
+      &--solid {
+        &.is-open {
+          margin-bottom: 22px;
+          border-bottom: 2px solid $c-gray-dcdcdc;
+        }
+      }
+      &.is-open {
+        &:focus{
+          -webkit-box-shadow: 0 0 0 1px #9bdaf1;
+          box-shadow: 0 0 0 1px #9bdaf1;
+        }
+        &:after{
+          transform: rotate(135deg);
+        }
+      }
+
+      &:after {
+        font-size: 2.5rem;
+        content: "+";
+        line-height: .65em;
+        overflow: hidden;
+        position: absolute;
+          right: 15px;
+          top: 25px;
+        transform-origin: center center;
+        transition: transform .2s ease;
+        color: $c-theme-secondary;
+      }
+    }
+
+    &__icon {
+      position: absolute;
+        top: 25px;
+        left: 20px;
+      text-align: center;
+
+      & > svg {
+        height: 25px;
+        width: 25px;
+        fill: $c-theme-secondary;
+      }
+    }
+
+    &__title {
+      @include ma-h5;
```

Comment:
why import both h4 and h5 styles?

Reply:
if you want to include the margin, I'd recommend only import h4 and manually set margin

Reply:
i think i took this from a pl scss - but i will pull it out